gravitee-gateway-core/src/main/java/io/gravitee/gateway/core/endpoint/lifecycle/GroupLifecycleManager.java

```diff
@@ -22,5 +22,5 @@
  * @author David BRASSELY (david.brassely at graviteesource.com)
  * @author GraviteeSource Team
  */
-public interface GroupLifecyleManager extends GroupManager, LifecycleComponent<GroupLifecyleManager> {
+public interface GroupLifecycleManager extends GroupManager, LifecycleComponent<GroupLifecycleManager> {
```

Comment:
it seems this refactoring breaks an unused import in `FreePlanApiHandlerConfiguration`